<?xml version="1.0"?>
<launch>

  <node pkg="control_bolide" type="stanley_controller.py" name="stanley_controller_node" output="screen"> 
    <!-- <param name="odom_topic" value=""/>  -->
    <param name="odom_topic" value="/pf/pos/odom"/> 

    <param name="waypoints_path" value="/home/bolide1/bolide_ws/src/course_2024_pkgs/control_bolide/racelines/esclangon_loop_2.csv"/> 
    <param name="cmd_topic" value="cmd_vel"/> 

    <param name="K_E" value="0.65"/> 
    <param name="K_H" value="0.45"/> 
    <param name="velocity_percentage" value="0.3"/>
    <param name="steering_limit" value="25.0"/>    
  </node> 

</launch>